source/_patterns/01-molecules/navigation/view-selector.yaml

```diff
@@ -4,15 +4,21 @@ properties:
   sideBySideUrl:
     type: string
     minLength: 1
-  articleUrl:
+  primaryUrl:
     type: string
     minLength: 1
-  figureUrl:
+  secondaryUrl:
     type: string
     minLength: 1
-  figureIsActive:
+  secondaryIsActive:
     type: boolean
     default: false
+  primaryViewSelector:
```

Comment:
This would be better as `PrimaryLabel` and will also need to be required.

Reply:
Done, primaryViewSelector is renamed to primaryLabel and it is required, secondaryViewSelector is renamed to secondaryLabel